app/src/main/java/com/door43/translationstudio/newui/home/RestoreFromCloudDialog.java

```diff
@@ -211,6 +211,7 @@ public boolean accept(File dir, String filename) {
                     }
 
                     TargetTranslationMigrator.migrateChunkChanges(AppContext.getLibrary(), targetTranslation);
+                    TargetTranslationMigrator.migrateFromUSXtoUSFM(targetTranslation.getId());
```

Comment:
Rather than calling these two migration methods we should call `TargetTranslationMigrator.migrate(sourcePath)` between lines 168 and 169.